autogluon/utils/try_import.py

```diff
@@ -82,3 +82,10 @@ def try_import_faiss():
         raise ImportError(
             "Unable to import dependency faiss"
             "A quick tip is to install via `pip install faiss-cpu`. ")
+
+def try_import_torch():
+    try:
+        import torch
+    except ValueError as e:
+        raise ImportError("Import torch failed.")
```

Comment:
Add to message: "Please install pytorch to use this functionality: `pip install torch`"

Reply:
Added to ImportError msg.